docs/developers.md

```diff
@@ -509,6 +509,28 @@ $ curl -H "Authorization: Bearer $(oc whoami -t)" -k https://asb-1338-ansible-se
 
 You should see a list of all bootstrapped specs and one that is labeled `localregistry-<bundle_name>`. I recommend using `|grep <bundle_name>` to help find it since the output is in JSON.
 
+### Alternative to using `apb run`
+Because of the limitations described above, it may be desired for a user to want the same functionality as `apb run` without having to rely on `apb push` being successful. This is because `apb run` implicitly performs `apb push` first before attempting to provision the application. In order to work around this, a user must first follow the above workaround to push their image onto the internal OpenShift registry. Once the image exists, you should be able to see the image with:
+```
+$ oc get images | grep <bundle_name>
```

Comment:
@dymurray As I described in bug [1541903](https://bugzilla.redhat.com/show_bug.cgi?id=1541903), I could NOT get the pushed image by querying the `bundle_name`, I think we should modify this command, or am I missing something?